### DISCUSSION TITLE
ColorPicker

### DISCUSSION THREAD

**Discussion by MasterProger** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2704] [Q&A] [answered]
When calling the DragDelta event on the Thump element, an error occurs if the Thump element is captured from the edge and dragged.
We need to remove the error.

**Comment by ElieTaillard** [comment, marked as answer]
If you have a bug it's may be more appropriate to open an new issue here: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues
